### DISCUSSION TITLE
404 NOT FOUND

### DISCUSSION THREAD

**Discussion by mallemagassa** [filamentphp/filament#5175] [Help]
Hi, I have an acceptance at the level of my resource when Button create its shows me a 404 NOT FOUND here is the capture below :

![Villes - Doucsoft - Google Chrome 05_12_2022 10_24_38 AM](https://user-images.githubusercontent.com/73896349/205661311-a1c91b89-82ad-466d-bd55-8b5f21056ff8.png)
![Villes - Doucsoft - Google Chrome 05_12_2022 10_24_53 AM](https://user-images.githubusercontent.com/73896349/205661861-ff034b79-4a22-48e9-a51a-0ffca00703dd.png)

**Comment by datlechin** [comment]
Your resource code?

**Comment by mallemagassa** [comment]
here is my resource code
![CityResource php - doucsoft - Visual Studio Code 05_12_2022 2_46_58 PM](https://user-images.githubusercontent.com/73896349/205666190-1cb0e621-36f9-4b54-ae33-46c9bcc3095f.png)
![CityResource php - doucsoft - Visual Studio Code 05_12_2022 2_47_20 PM](https://user-images.githubusercontent.com/73896349/205666196-82a8ced3-543b-46dc-9c6b-95055aadd69e.png)
